### PR TITLE
Several updates

### LINKED ISSUE #73: Verify accessibility standards
**What needs to be done? Why does it need to be done?**
With all of the V2 functionality in place, verify the accessibility standards of the app, as described in the "Accessibility" section of the V2 PRD.

**Additional context**
See the "Accessibility" section of the V2 PRD.

### LINKED ISSUE #81: Add formatting for numbers in UI
**Is your feature request related to a problem? Please describe.**
Currently the UI prints plain numbers without any formatting, e.g. '11428' '336', and '35455' in the attached image. Of these, '336' looks fine, but the other two longer numbers look unusual to a casual user.

image

**Describe the solution you'd like**
Numbers larger than 3 digits should have commas every 3 digits to match en-US locale. For example:
11428 --> 11,428
35455 --> 35,455

Here are other examples that aren't in the screenshot:
2345678 --> 2,345,678
7653% --> 7,653%

**Additional context**
This may be easier to implement with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat or the older https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Be careful not to accidentally add decimal units - no numbers in the tool should have decimals.

### LINKED ISSUE #82: Update impact allocations (% from new admissions, % from revocations) in UI
**Is your feature request related to a problem? Please describe.**
As discussed on  https://github.com/Recidiviz/supervision-success-component/pull/79, we currently show the breakout of allocations for both fiscal and human impact. But these numbers are always the same for both - so the user experience will feel a little wonky. 

**Describe the solution you'd like**
We should design a very small revision to de-duplify these and change the placement slightly. @agronomic could you add a high-fidelity mock here when available?

**Describe alternatives you've considered**
Here's an example of a lightweight change to address this:
image

### LINKED ISSUE #83: Minor copy updates for sliders
**Is your feature request related to a problem? Please describe.**
The sliders currently have the following copy:
image

Specifically, the line "**336** Violations resulting in Yuta <line break> incarceration" has the following issues:

1. 'Violations' should be lower-case (so it should be: "**336** violations resulting...")
2. There should not be a word 'Yuta' (so it should be: "...resulting in incarceration")
3. There should not be a line break

Separately, the line "**35455** New admissions to prison" has the following issue:

1. 'New' should be lower-case (so it should be: "**35455** new admissions...")

**Describe the solution you'd like**
These two lines should look like this, instead:

- **336** violations resulting in incarceration
- **35455** new admissions to prison

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

1. Minor copy updates for sliders
2. Update impact allocations (% from new admissions, % from revocations) in UI
3. Add formatting for numbers in UI
4. Verify accessibility standards 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #73 , closes #81 , closes #82 , closes #83 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/SupervisionSuccess/utils/formatNumber.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+const formatNumber = (number, isAbsolute = false) => {
```
Comment (jessex) [inline src/SupervisionSuccess/utils/formatNumber.js:17]: Let's add a note about what exactly this changes, i.e. what formatting it does.

**Review by jessex — CHANGES_REQUESTED** [review]
Nice batch, @nasaownsky! A few issues from local testing:

Regarding #73, if you go into Chrome's developer tools and click "Lighthouse" and generate a report, Accessibility is currently getting a 98/100, which is great! But there is one small issue still outstanding that we should fix:
Screen Shot 2021-07-22 at 14 20 50

Regarding #81, the issue calls out that we want to make sure we don't introduce any decimals into our numbers, but it looks like we did in this PR. Now, if the costs are below $100M, it will add a decimal to keep using 3 digits. We don't want that to happen. The number below should just round off to $82M:
Screen Shot 2021-07-22 at 14 13 00

Regarding #82, if you look at this in mobile, the new section has some kind of strange padding. It looks further away from the outcomes numbers it corresponds to, and closer to the title of the chart section. We should make it so that it's at least not that close to the chart text:
Screen Shot 2021-07-22 at 14 17 35

Also, @nasaownsky, I noticed in that mobile screenshot that the chart is behaving strangely, like it's cut off horizontally maybe? It's not displaying all of the x-axis values you would expect. Can you look into that? Test it out with different projection length values and make sure you can actually see the entire chart on mobile.

**Comment by nasaownsky** [comment]
@jessex Fixed all you have requested. The issue with chart happens when xAxis ticks don't fit in available space. The data though shows correctly. I can decrease font-size, but on some scales all ticks won't be able to show. This is how chart.js works.

Снимок экрана 2021-07-23 в 21 08 00

**Comment by jessex** [comment]
Thanks for this, @nasaownsky! I can see that getting the x-axis labels perfectly in frame for all possible viewports is a challenge given our current setup. The only unfortunate thing about it at the moment is that when you test out different resolutions/devices, you see that x-axis values will be missing for certain projection lengths. For example, for Pixel 2XL, you see that there are labels missing when projection length is 4 years but not when it's 3. Conversely, for Galaxy S5, the opposite is true, and labels are missing only when projection length is 3. I am going to file a separate ticket to investigate it, but we don't need to block this work on that.

**Review by jessex — APPROVED** [review]
The previously noted issues are all fixed. Good to go on this one!

**Pull request merged**